Scripts/RemediationScripts/Remediate-DisableRemoteDebuggingForAppServices.ps1

```diff
@@ -167,23 +172,36 @@ function Disable-RemoteDebuggingForAppServices
 
         [String]
         [Parameter(ParameterSetName = "WetRun", HelpMessage="Specifies the path to the file to be used as input for the remediation")]
-        $FilePath
+        $FilePath,
+
+        [String]
+        $Path,
```

Comment:
Add parameter description and parameter set name etc.

Reply:
As you added this to WetRun parameter set, make sure while running individual BRS this should not ask user to provide value for 'path' parameter. Plz test this once, if not already done.

Reply:
As it is not mandatory, It won't ask user to provide these parameters